shell/server/proxy.js

```diff
@@ -1642,6 +1642,9 @@ class Proxy {
     } else if ('noContent' in rpcResponse) {
       const noContent = rpcResponse.noContent;
       const noContentCode = noContentSuccessCodes[noContent.shouldResetForm * 1];
+      if (noContent.eTag) {
+        response.setHeader('ETag', composeETag(noContent.eTag));
+      }
```

Comment:
Our javascript linter is requesting that you put a blank line after this closing brace. You can run the linter locally by doing `make stylecheck`.